### DISCUSSION TITLE
[讨论] MCIM 目前是否足够稳定？

### DISCUSSION THREAD

**Discussion by SALTWOOD** [Meloong-Git/PCL#5102] [讨论]
前几天调试，看到这个选项
![暂时无效](https://github.com/user-attachments/assets/4a0f8bf5-8c82-4ad3-b97e-428b7302e4b8)

作为 OpenMCIM 的后端维护者之一，我看到每天都有很多的请求到我写的后端，而这甚至还是在“仅在官方源加载缓慢时”的请求量。
因此我想询问一下各位，据你们最真实的用户体验，MCIM 的稳定性如何？你们的回答将有助于我们改善 MCIM 及 OpenMCIM。

**Comment by ZeroWolf233** [comment]
或许以现在的全部带宽(所有在线节点都正常的话)，可能也扛不住PCL的流量我感觉

**Reply by SALTWOOD** [reply]
但是现在根本没有几个节点……只有 AKIAM 的反代啥的
而且默认策略也是平衡模式，除非全都切成镜像源否则是不会炸的

**Comment by LTCatt** [comment]
刚才测了一下，随便选了一个整合包，192 个 Mod 中，有 44 个无法从 MCIM 源获取到正确的信息。
如果打开了调试模式并修改下载源设置，就会优先使用 MCIM 源，此时在 Mod 管理页面刷新，没有图标的就是获取信息失败的。

---

Create A&B
（最上面几个 mod 是我自己下的）

官方源优先，检测到 89 个可更新 Mod：
![image](https://github.com/user-attachments/assets/70b5745c-a900-46f7-9e70-a83735889ef5)

MCIM 源优先，检测到 16 个可更新 Mod：
![image](https://github.com/user-attachments/assets/89bf215f-bdc9-49dd-bb7e-9c0f359f3c50)

**Reply by SALTWOOD** [reply]
行罢……我去跟 z0z0r4 说一声，这个算是 MCIM 的部分不是我管的 OpenMCIM（

**Comment by ZeroWolf233** [comment]
测试了100次请求
这是完整log: [论稳定性.log](https://github.com/user-attachments/files/17691273/default.log)
![310b2b6c72c2d77818de28a36924f22e](https://github.com/user-attachments/assets/733bce2f-333c-425b-8aa6-cf445cecc47a)
自己看罢...

**Comment by wuliaodexiaoluo** [comment]
就目前来看，稳定性不佳....

**Reply by ZeroWolf233** [reply]
正确的

**Reply by WorldHim** [reply]
中肯的
